src/components/features/Brand/Brand.js

```diff
@@ -33,25 +36,45 @@ class Brand extends React.Component {
     let currentPage = this.state.activePage;
     const { brands, rwdMode } = this.props;
 
-    const pagesCount = Math.ceil(brands.length / this.rwdCardsInRow[rwdMode.rwdMode]);
+    const pagesCount = Math.ceil(brands.length / this.rwdBrandsInRow[rwdMode.rwdMode]);
 
     if (currentPage < pagesCount - 1) {
       this.handlePageChange(currentPage + 1);
     }
   }
 
   changePageNext() {
-    let currentBrand = this.state.activePage;
+    let currentPage = this.state.activePage;
+
+    if (currentPage !== 0) {
+      this.handlePageChange(currentPage - 1);
+    }
+  }
 
-    if (currentBrand !== 0) {
-      this.handlePageChange(currentBrand - 1);
+  handleChangeBrandLine() {
+    let changingButtons = this.state.activePage;
```

Comment:
Dlaczego let? 
Nie rozumiem tego kodu, tłumacząc to na polski to aktywna strona jest zapisywana jako zmieniające się buttony, któraś nazwa jest nie poprawna.

Reply:
sorry, zapomniałem posprzątać ;)